server/server.js

```diff
@@ -2,6 +2,8 @@
 const express = require("express");
 const cors = require('cors')
 const app = express();
+require('dotenv').config();
```

Comment:
Do we use this?

Reply:
yes, there is an env that i am using in the server